search/query.go

```diff
@@ -40,29 +40,29 @@ func createResultsQuery(resultTypes []string, q Query, docIds []string) elastic.
 			// Don't calculate score here, as we use sloped score below.
 			elastic.NewConstantScoreQuery(
 				elastic.NewBoolQuery().Should(
-					elastic.NewMatchQuery("title.language", q.Term),
-					elastic.NewMatchQuery("description.language", q.Term),
-					elastic.NewMatchQuery("content.language", q.Term),
+					elastic.NewMatchQuery("title.language", q.Term).Analyzer("synonym"),
+					elastic.NewMatchQuery("description.language", q.Term).Analyzer("synonym"),
+					elastic.NewMatchQuery("content.language", q.Term).Analyzer("synonym"),
 				).MinimumNumberShouldMatch(1),
 			).Boost(0.0),
 		).Should(
 			elastic.NewDisMaxQuery().Query(
 				// Language analyzed
-				elastic.NewMatchPhraseQuery("title.language", q.Term).Slop(SLOP).Boost(TITLE_BOOST),
-				elastic.NewMatchPhraseQuery("description.language", q.Term).Slop(SLOP).Boost(DESCRIPTION_BOOST),
-				elastic.NewMatchPhraseQuery("content.language", q.Term).Slop(SLOP),
+				elastic.NewMatchPhraseQuery("title.language", q.Term).Analyzer("synonym").Slop(SLOP).Boost(TITLE_BOOST),
```

Comment:
Does this defines the language?
How that works for different languages?
I thought the analyzer was chosen per field?
Please explain?

is this query time analyzer? which in different from index time?